pages/api/v1/goalies/ratings/[id].ts

```diff
@@ -0,0 +1,73 @@
+import Cors from 'cors';
+import { NextApiRequest, NextApiResponse } from 'next';
+import SQL from 'sql-template-strings';
+
+import { query } from '../../../../../lib/db';
+import use from '../../../../../lib/middleware';
+
+const cors = Cors({
+  methods: ['GET', 'HEAD'],
+});
+
+export default async (
+  req: NextApiRequest,
+  res: NextApiResponse
+): Promise<void> => {
+  await use(req, res, cors);
+
+  const { league = 0, season: seasonid, id } = req.query;
+
+  const basePlayerData = await query(SQL`
+  SELECT *
+  FROM corrected_player_ratings
+  INNER JOIN player_master
+  ON corrected_player_ratings.PlayerID = player_master.PlayerID
+  AND corrected_player_ratings.SeasonID = player_master.SeasonID
+  AND corrected_player_ratings.LeagueID = player_master.LeagueID
+  INNER JOIN team_data
+  ON player_master.TeamID = team_data.TeamID
+  AND corrected_player_ratings.SeasonID = team_data.SeasonID
+  AND corrected_player_ratings.LeagueID = team_data.LeagueID
+  WHERE corrected_player_ratings.LeagueID=${+league}
+  AND corrected_player_ratings.G=20
+  AND player_master.TeamID>=0
+  AND corrected_player_ratings.PlayerID = ${+id}
```

Comment:
this is the only change.  Removes the season from the where statement (unless it is requested) and add player id to the where statement.